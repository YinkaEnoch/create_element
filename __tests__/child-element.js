const c = require("../src/index.ts");

test("add items to an unordered list", () => {
	let el = c("ul", { id: "state-of-origin" }, [
		c("li", { class: "grey-text" }, "Oyo"),
		c("li", {}, "Osun"),
		c("li", {}, "Ogun"),
		c("li", {}, "Ondo"),
	]);

	expect(el).toMatchSnapshot();
});
