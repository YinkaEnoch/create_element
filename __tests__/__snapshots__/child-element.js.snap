// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`add items to an unordered list 1`] = `
<ul
  id="state-of-origin"
>
  <li
    class="grey-text"
  >
    Oyo
  </li>
  <li>
    Osun
  </li>
  <li>
    Ogun
  </li>
  <li>
    Ondo
  </li>
</ul>
`;
